frontend/src/components/@common/Carousel/Carousel.stories.tsx

```diff
@@ -0,0 +1,40 @@
+
+const meta = {
+  title: 'common/Carousel',
+  component: Carousel,
+} satisfies Meta<typeof Carousel>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+const pngItems = [
+  { type: 'image/png', src: testStartImage },
+  { type: 'image/png', src: donggleExample },
+  { type: 'image/png', src: testImage },
+  { type: 'image/png', src: testImage2 },
+  { type: 'image/png', src: testLastImage },
+] as Media[];
+
+export const Primary: Story = {
+  args: {
+    medias: pngItems,
+  },
+  argTypes: {
+    medias: {
+      description: '캐러셀 내부에 보여질 미디어 컨텐츠 배열입니다.',
+    },
+  },
+};
```

Comment:
```suggestion
export const Primary: Story = {
  args: {
    medias: pngItems,
  },
  argTypes: {
    medias: {
      description: '캐러셀 내부에 보여질 미디어 컨텐츠 배열입니다.',
    },
    width: {
      options: ['400px', '600px', '800px', '1000px'],
      control: {
        type: 'radio',
      },
    },
    height: {
      options: ['400px', '600px', '800px', '1000px'],
      control: {
        type: 'radio',
      },
    },
  },
};
```
현재 스토리에 width, height 값을 넣으면 렉이 걸리면서 캐러셀이 깨지는데, 미리 옵션을 지정해서 라디오 버튼으로 보여주니 잘 동작하더라구요. 이렇게 제안해봅니다.

Reply:
커밋 적용했어요!